### PR TITLE
update div and hbp for subplot renaming

### PR DESCRIPTION
**Pull request by gitbarnitt**
Hi Claire, sorry for the delay on these.  I did confirm that other relevant quick start guides don't have old subplot terminology.  In some cases figures don't match latest dpugs, but if they were not wrong, I did not update.

### CODE REVIEW

File: DP1.10058.001/Image.legend1.txt [hunk 1]
```diff
@@ -1 +1 @@
...
+The square, multi-scale plot used to record plant species composition and cover. The plot includes nested subplots at specific locations within the plot. The 100m^2 subplot naming (e.g., 31_100, 32_100, 40_100, 41_100) corresponds to the identity of the point in the southwest corner of the subplot. Smaller subplots (i.e., 1m^2 and 10m^2) are named according to the 100m^2 subplots within which they are nested, the subplot scale, and the corner of the 100m^2 subplot; corners within the 100m^2 subplot are numbered counterclockwise starting in the southwest corner. Subplot 32_1_2, for example, is the 1m^2 subplot in corner 2 of 100m^2 subplot 32_100. Prior to 2019, there were additional subplots near the center of the plot: 31_1_4, 31_10_4, 41_1_1, and 41_10_1 that were discontinued.
```
Comment (cklunch) [inline DP1.10058.001/Image.legend1.txt:1]: Is there a document we can link to here? Something like "For full details on subplot naming conventions, see:"

File: DP1.10058.001/Table.joining.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 |Table 1|Table 2|Join by field(s)|
 |------------------------|------------------------|-------------------------------|
 div_1m2Data|div_10m2Data100m2Data|Direct join not recommended. See Standard calculations.
-div_geneticarchive|Any other table|Join not recommended. Data resolution does not match other tables.
+div_1m2Data|div_morphospecies|siteID and morphospeciesID
```
Comment (cklunch) [inline DP1.10058.001/Table.joining.md:4]: Needs to be siteID, morphospeciesID to be read correctly by neonOS.

File: DP1.10058.001/Standard.calculations.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
-Species-specific cover observed in 1m^2 subplots is directly available in the table div_1m2Data. Subplots at which observations were made are identified by the subplotID field (e.g., 31.1.1, 40.3.1), and are associated with specific plots (identified d by the plotID field, e.g., HARV_001) within the site (Figure 1).  
+Species-specific cover observed in 1m^2 subplots is directly available in the table div_1m2Data. Subplots at which observations were made are identified by the subplotID field (e.g., 31_1_1, 40_1_3), and are associated with specific plots (identified by the plotID field, e.g., HARV_001) within the site (Figure 1).
```
Comment (cklunch) [inline DP1.10058.001/Standard.calculations.txt:1]: Add a link to neonPlants to the Standard calculations section as well?

**Review by cklunch — COMMENTED** [review]
@gitbarnitt Thanks! Just a few suggestions on the edits.

File: DP1.10058.001/Image.legend1.txt (hunk 1 above)
Comment (gitbarnitt) [reply DP1.10058.001/Image.legend1.txt:1]: added: "see NEON.DOC.005108 NEON Terrestrial Observation System: Spatial and Temporal Sampling Strategy, and NEON.DOC.014042 TOS Protocol and Procedure: DIV - Plant Diversity Sampling."

File: DP1.10058.001/Table.joining.md (hunk 2 above)
Comment (gitbarnitt) [reply DP1.10058.001/Table.joining.md:4]: I had neglected to add the new(ish) identificationHistory table.  With that, I removed the "no other tables" lines for the genetic archive and voucher

File: DP1.10058.001/Standard.calculations.txt (hunk 3 above)
Comment (gitbarnitt) [reply DP1.10058.001/Standard.calculations.txt:1]: done

Comment (gitbarnitt) [reply DP1.10058.001/Standard.calculations.txt:1]: @cklunch edits made, thanks for the good suggestions!

**Pull request merged**